openmc/universe.py

```diff
@@ -519,3 +494,41 @@ def create_xml_subelement(self, xml_element):
                 # Append the Universe ID to the subelement and add to Element
                 cell_element.set("universe", str(self._id))
                 xml_element.append(cell_element)
+
+    def _determine_paths(self, path=''):
+        """Count the number of instances for each cell in the universe, and
+        record the count in the :attr:`Cell.num_instances` properties."""
+
+        univ_path = path + 'u{}'.format(self.id)
+
+        for cell in self.cells.values():
```

Comment:
I presume that you verified that the cells are iterated over in the same order here as they are in the Fortran code?